.gitlab-ci.yml

```diff
@@ -44,34 +45,28 @@ variables:
 build-and-test:
   stage: build
   <<: *docker-env
+  variables:
+    SUBSTRATE_PATH:                "./substrate"
   before_script:
     # upstream CI can override global vars here
     - unset CARGO_TARGET_DIR
+    - rustup +nightly show
+    - cargo --version
+    - yarn --version
+    - npm --version
+    - wasm-prune -V
+    - sccache -s
     # `cargo install` returns an error if there is nothing to update, so have to supress it here temporarily
     - cargo contract -V
-    - cargo install --git https://github.com/paritytech/cargo-contract || echo $?
+    - cargo install --git https://github.com/paritytech/cargo-contract --branch master || echo $?
```

Comment:
Can we instead use fixed releases?

Reply:
of course, it's just not yet decided where to store them => where to take them from